src/index.js

```diff
@@ -276,6 +276,9 @@ export function strRepeat(num: any, bit: number) {
 
 // video 时间格式化
 export function formatTime(time: number) {
+  if (!window.isFinite(time)) {
```

Comment:
Remove `window` here. It may be run in a non-browser environment.
```suggestion
  if (!isFinite(time)) {
```